pkg/profile/v1-profile.json

```diff
@@ -1,164 +1,184 @@
         "bodyShape": {
-          "type": "string",
-          "minLeght": 1,
-          "maxLeght": 100
-        },
-        "skinColor": {
           "type": "object",
           "properties": {
-            "r": {
-              "type": "number",
-              "minimum": 0,
-              "maximum": 1
-            },
-            "g": {
-              "type": "number",
-              "minimum": 0,
-              "maximum": 1
+            "contentName": {
+              "type": "string"
             },
-            "b": {
-              "type": "number",
-              "minimum": 0,
-              "maximum": 1
+            "contents": {
+              "type": "array",
+              "items": {
+                "type": "object",
+                "properties": {
+                  "file": {
+                    "type": "string"
+                  },
+                  "hash": {
+                    "type": "string"
+                  }
+                },
+                "required": ["file", "hash"]
+              }
             }
           },
-          "additionalProperties": false,
-          "required": [ "r", "g", "b" ]
+          "required": ["contentName", "contents"]
         },
-        "hairColor": {
+        "eyes": {
           "type": "object",
           "properties": {
-            "r": {
-              "type": "number",
-              "minimum": 0,
-              "maximum": 1
+            "mask": {
+              "type": "string"
             },
-            "g": {
-              "type": "number",
-              "minimum": 0,
-              "maximum": 1
+            "texture": {
+              "type": "string"
             },
-            "b": {
-              "type": "number",
-              "minimum": 0,
-              "maximum": 1
+            "color": {
+              "type": "object",
+              "properties": {
+                "r": {
+                  "type": "number"
+                },
+                "g": {
+                  "type": "integer"
+                },
+                "b": {
+                  "type": "number"
+                }
+              },
+              "required": ["r", "g", "b"]
             }
           },
-          "additionalProperties": false,
-          "required": [ "r", "g", "b" ]
+          "required": ["mask", "texture", "color"]
         },
-        "eyeColor": {
+        "mouth": {
           "type": "object",
           "properties": {
-            "r": {
-              "type": "number",
-              "minimum": 0,
-              "maximum": 1
+            "mask": {
+              "type": "string"
             },
-            "g": {
-              "type": "number",
-              "minimum": 0,
-              "maximum": 1
-            },
-            "b": {
-              "type": "number",
-              "minimum": 0,
-              "maximum": 1
+            "texture": {
+              "type": "string"
             }
           },
-          "additionalProperties": false,
```

Comment:
by removing addionalProperties on everything you are allowing anything, I don't think that's a good idea